pulsar-broker-common/src/main/java/org/apache/pulsar/broker/authorization/PulsarAuthorizationProvider.java

```diff
@@ -126,7 +125,7 @@ public void initialize(ServiceConfiguration conf, ConfigurationCacheService conf
                             PulsarServerException ex = new PulsarServerException(
                                     String.format("%s is not authorized to access subscription %s on topic %s", role,
                                             subscription, topicName));
-                            permissionFuture.complete(false);
+                            permissionFuture.completeExceptionally(ex);
```

Comment:
Good catch. I think the exception is useless here. We should be completing the future with `false` and avoid creating the exception altogether.

Reply:
ok, I will fix it.